packages/fts-http/src/handler.ts

```diff
@@ -52,6 +52,15 @@ export function createHttpHandler(
           args.push(context)
         }
 
+        // Push additional props into args if allowing additionalProperties
+        if (definition.params.schema.additionalProperties) {
+          Object.keys(params).forEach(
+            (name) =>
+              definition.params.order.indexOf(name) === -1 &&
+              args.push([name, params[name]])
```

Comment:
i find it a general best practice to avoid using expressions for side effects. in this case, using an `if` statement would make it more clear what's going on here.

Reply:
Addressed

Reply:
Just thinking, this isn't going to be covered by fixtures. Will need to add.